tests/sentry/ingest/ingest_consumer/test_ingest_consumer_processing.py

```diff
@@ -195,7 +195,18 @@ def test_with_attachments(default_project, task_runner, missing_chunks, monkeypa
     "event_attachments", [True, False], ids=["with_feature", "without_feature"]
 )
 @pytest.mark.parametrize(
-    "chunks", [(b"Hello ", b"World!"), (b"",), ()], ids=["basic", "zerolen", "nochunks"]
+    "chunks",
+    [
+        {"payload": (b"Hello ", b"World!")},
+        {"payload": (b"",)},
+        {"payload": ()},
+        {
+            "type": "event.view_hierarchy",
+            "content-type": "application/json",
+            "payload": (b'{"rendering_system":"flutter","windows":[]}',),
```

Comment:
This test parametrization confused me because I thought `payload` was an actual field of the attachment. Should we parametrize with a tuple `"attachment_type,chunks,content_type"` instead?